### PR TITLE
e2e cleanup - Lint playbooks (volume-policy tests)

### PR DESCRIPTION
**Pull request by vijaysinghrawat1**
Signed-off-by: vijaysinghrawt1 <vijaysinghrawat167@gmail.com>

**What this PR does / why we need it**:
**Issue Type:**
   Ansible-lint errors
   Trailing whitespace
**Ansible Version:**
ansible 2.5..0
**Ansible Configuration:**
None

**Environment**:
N/A

**Summary**:
fixing the errors regarding ansible-lint testing.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
[ANSIBLE0002] Trailing whitespace.
[ANSIBLE0012] Commands should not change things if nothing needs doing.
[ANSIBLE0013] Use shell only when shell functionality is required.
[ANSIBLE0011] All tasks should be named.

**Special notes for your reviewer**:
@ksatchit  Sir please review.

### CODE REVIEW

File: e2e/ansible/playbooks/hyperconverged/test-k8s-volume-taints/vol-policies.yaml [hunk 1]
```diff
@@ -35,6 +35,9 @@ spec:
   - name: ReplicaImage
     value: openebs/jiva:0.5.0
   - name: ReplicaCount
+|1|mMWrHaeuj5Fh1uA5z/qaj1mlR1c=|/0fHJ7v7lAr3bFTQsSQiv1I/CAY= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBH7cWMT0QWKE2ejgK6d8jYLG4L+MkDvooidBfMLS8Uxw2zC66b33phoVt/WziScyn8kSWrFFbTwTAHjw1mARA0Q=
+|1|A56+E/F9GdJTvtqHXl6i8Ouq/sE=|nq23O1MZfc/QqaYi5smqV0PnW88= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBN2XR2bCP/xNyXlFtdIXFM0TD0dbINPtHGpr3Lka/+JwGMzyyampDaZjanduuUQfabTYNCEHq4OnioghP9b/bjc=
+|1|TXze/2804v0mqwwNHZKiHM9GKss=|2wKhErVljvO9eoA2jYYVWq15vvU= ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEqGgSg03pBjEUXV/dnF0Xi9Kkc2FrwIM+6L+GPARr9mBlh7LizJkc/TV3DCEpmEtEFMGDdaU+JEfcS6SyjrDEg=
```
Comment (ksatchit) [inline e2e/ansible/playbooks/hyperconverged/test-k8s-volume-taints/vol-policies.yaml]: Is this intended ?

File: e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yaml [hunk 2]
```diff
@@ -148,27 +148,27 @@
...
        - name: 8) Get pod name
          shell: kubectl get pods --all-namespaces -o wide | grep "{{ pv_name.stdout }}" | grep rep | grep "{{ node1.stdout }}" | awk '{print $2}'
          args:
            executable: /bin/bash
-         register: pod_name 
+         register: pod_name
 
        - name: 8a) Delete the replica on one of the nodes
-         shell: kubectl delete pod "{{ pod_name.stdout }}" --grace-period=0 --force
+         command: kubectl delete pod "{{ pod_name.stdout }}" --grace-period=0 --force
```
Comment (ksatchit) [inline e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yaml]: Does the linter recommend replacing shell with command ? If so, what is the criteria ?

Comment (vijaysinghrawat1) [reply e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yaml]: Thanks for asking,
Yes Linter recommended to replace shell.
 The shell module is potentially more dangerous compare to command module (ok, nothing is really stopping you doing command: rm -rf --no-preserve-root) and should only be used when you actually need shell functionality. 
If you’re not stringing two commands together (using pipes or even just && or ;), you don’t really need the shell module. Similarly, expanding shell variables or file globs require the shell module. If you’re not using these features, don’t use the shell module. If you are using these features, think twice if you can rewrite the shell command to make it more Ansible friendly.

File: e2e/ansible/playbooks/hyperconverged/test-k8s-volume-taints/vol-policies.yaml (hunk 1 above)
Comment (vijaysinghrawat1) [reply e2e/ansible/playbooks/hyperconverged/test-k8s-volume-taints/vol-policies.yaml]: No,

File: e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yaml (hunk 2 above)
Comment (ksatchit) [reply e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yaml]: In the CI, we are sourcing the profile before running each kubectl command. Can you confirm if the current playbook can run successfully in the CI env ?

**Pull request closed without merging**